From 9db32c3508a8c5d357a053d5f8278c34b4df18f3 Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <akitouni@gnome.org>
Date: Sun, 12 Feb 2023 13:13:21 +0100
Subject: [PATCH] build: accept both babl and babl-0.1

The latest release of babl is named babl, while the latest git head is named
babl-0.1

Fixes #208
---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 7fa36f5f..a0bd204b 100644
--- a/meson.build
+++ b/meson.build
@@ -4,7 +4,7 @@ project(
   version: '43.0',
   license: 'GPL3+',
   default_options: 'buildtype=debugoptimized',
-  meson_version: '>= 0.50.0',
+  meson_version: '>= 0.60.0',
 )
 
 photos_prefix = get_option('prefix')
@@ -152,7 +152,7 @@ libgd = subproject(
 )
 libgd_dep = libgd.get_variable('libgd_dep')
 
-babl_dep = dependency('babl-0.1')
+babl_dep = dependency('babl-0.1', 'babl')
 cairo_dep = dependency('cairo', version: '>= 1.14.0')
 gdk_pixbuf_dep = dependency('gdk-pixbuf-2.0', version: '>= 2.36.8')
 gegl_dep = dependency('gegl-0.4', version: '>= 0.4.0')
-- 
GitLab

